build/rpm/dependencies.yaml

```diff
@@ -168,14 +168,23 @@
     'pupmod-razorsedge-snmp': '3.9.0-0'
 
 # Camptocamp has assumed ownership of the systemd project
+# Subsequently, Voxpupuly assumed ownership of the systemd project
 'systemd':
   :obsoletes:
     'pupmod-simp-systemd': '2.1.0-0'
+    'pupmod-camptocamp-systemd': '2-12.0-1'
+
+# Out of maintenance, trying to get re-opened
```

Comment:
Looks like it's been re-opened and receiving PRs: https://github.com/puppetlabs/puppetlabs-puppet_authorization